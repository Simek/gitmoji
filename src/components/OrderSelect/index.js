// @flow
import React from 'react'

type Props = { onChange: function }

const OrderSelect = (props: Props) => (
    <div className="order-select-wrapper">
        <ins>ðŸ”€</ins>
        <select
            className="order-select order-select-pink"
            onChange={event => props.onChange(event.target.value)}
        >
            <option value="default">Default order</option>
            <option value="name">Emoji name</option>
            <option value="description">Description</option>
        </select>
    </div>
)

export default OrderSelect
