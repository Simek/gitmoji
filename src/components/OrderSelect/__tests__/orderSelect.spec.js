import renderer from 'react-test-renderer'

import OrderSelect from '../index'
import * as stubs from './stubs'

describe('OrderSelect', () => {
  it('should render the component', () => {
    const wrapper = renderer.create(<OrderSelect {...stubs.props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
