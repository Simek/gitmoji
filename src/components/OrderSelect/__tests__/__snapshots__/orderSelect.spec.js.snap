// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderSelect should render the component 1`] = `
<div
  className="order-select-wrapper"
>
  <ins>
    ðŸ”€
  </ins>
  <select
    className="order-select order-select-pink"
    onChange={[Function]}
  >
    <option
      value="default"
    >
      Default order
    </option>
    <option
      value="name"
    >
      Emoji name
    </option>
    <option
      value="description"
    >
      Description
    </option>
  </select>
</div>
`;
